hudi-aws/pom.xml

```diff
@@ -268,11 +269,12 @@
                                     </run>
                                 </image>
                                 <image>
-                                    <name>motoserver/moto:${moto.version}</name>
+                                    <name>apachehudi/moto:${moto.version}</name>
```

Comment:
We have to do this to pull the image of the correct architecture as the plugin cannot do this properly.